@here/harp-mapview/lib/TileProxy.ts

```diff
@@ -0,0 +1,268 @@
+ * is simple to offset the geometry (the THREE.Object3D must be cloned and then shifted,
+ * unfortunately this is the only way to render a given mesh / material twice, a THREE.Object3D is
+ * however quite lightweight so this doesn't pose any problem).
+ */
+export class TileProxy implements ITile {
+    /**
+     * Copies of the objects in the proxied [[Tile]].
+     */
+    objects: TileObject[] = [];
+
+    /**
+     * @inheritdoc
+     *
+     * Used to track the proxied [[Tile]]s geometry version (to know when it has changed so that the
+     * objects can be cloned).
+     */
+    geometryVersion: number = 0;
+
+    get geoBox(): GeoBox {
+        return this.tile.geoBox;
+    }
+
+    get boundingBox(): THREE.Box3 {
+        return this.tile.boundingBox;
+    }
+
+    get center(): THREE.Vector3 {
+        return this.tile.center;
+    }
+
+    get placedTextElements(): GroupedPriorityList<TextElement> {
+        return this.tile.placedTextElements;
+    }
+
+    get memoryUsage(): number {
+        return this.tile.memoryUsage;
+    }
+
+    get usageStatistics(): TileResourceUsageInfo {
+        return this.tile.usageStatistics;
+    }
+
+    get userTextElements(): TextElement[] {
+        return this.tile.userTextElements;
+    }
+
+    get textElementGroups(): GroupedPriorityList<TextElement> {
+        return this.tile.textElementGroups;
+    }
+
+    get disposed(): boolean {
+        return this.tile.disposed;
+    }
+
+    get hasGeometry(): boolean {
+        return this.tile.hasGeometry;
+    }
+
+    get dataSource(): DataSource {
+        return this.tile.dataSource;
+    }
+
+    get tileKey(): TileKey {
+        return this.tile.tileKey;
+    }
+
+    get textElementsChanged(): boolean {
+        return this.tile.textElementsChanged;
+    }
+
+    set textElementsChanged(val: boolean) {
+        this.tile.textElementsChanged = val;
+    }
+
+    get roadIntersectionData(): RoadIntersectionData | undefined {
+        return this.tile.roadIntersectionData;
+    }
+
+    get copyrightInfo(): CopyrightInfo[] | undefined {
+        return this.tile.copyrightInfo;
+    }
+
+    set copyrightInfo(val: CopyrightInfo[] | undefined) {
+        this.tile.copyrightInfo = val;
+    }
+
+    get frameNumRequested(): number {
+        return this.tile.frameNumRequested;
+    }
+
+    get frameNumVisible(): number {
+        return this.tile.frameNumVisible;
+    }
+
+    set frameNumVisible(val: number) {
+        this.tile.frameNumVisible = val;
+    }
+
+    get frameNumLastVisible(): number {
+        return this.tile.frameNumLastVisible;
+    }
+
+    set frameNumLastVisible(val: number) {
+        this.tile.frameNumLastVisible = val;
+    }
+
+    get numFramesVisible(): number {
+        return this.tile.numFramesVisible;
+    }
+
+    set numFramesVisible(val: number) {
+        this.tile.numFramesVisible = val;
+    }
+
+    get isVisible(): boolean {
+        return this.tile.isVisible;
```

Comment:
Sharing `isVisible` and `frameNumVisible` is not good idea, as we end-up with 
* `isVisible` but not actually visible, because out-of-frustum "parent Tiles"
* `IsVisible` but not actually visible and evictable! proxies, because some of it's parent proxies is visible...

This `isVisible` should be either renamed or well documented.